led_strip/examples/led_strip_rmt_ws2812/main/led_strip_rmt_ws2812_main.c

```diff
@@ -11,21 +11,17 @@
 #include "esp_err.h"
 
 // GPIO assignment
-#define LED_STRIP_BLINK_GPIO  8
+#define LED_STRIP_BLINK_GPIO  14
 
 // LED numbers in the strip
-#define LED_STRIP_LED_NUMBERS 1
-
-// 10MHz resolution, 1 tick = 0.1us (led strip needs a high resolution)
-#define LED_STRIP_RMT_RES_HZ  (10 * 1000 * 1000)
+#define LED_STRIP_LED_NUMBERS 24
 
 static const char *TAG = "example";
+#define LED_STRIP_RMT_RES_HZ  (10 * 1000 * 1000)
+// 10MHz resolution, 1 tick = 0.1us (led strip needs a high resolution)
```

Comment:
reorder the definition and comments
```suggestion
// Numbers of the LED in the strip
#define LED_STRIP_LED_NUMBERS 24
// 10MHz resolution, 1 tick = 0.1us (led strip needs a high resolution)
#define LED_STRIP_RMT_RES_HZ  (10 * 1000 * 1000)

static const char *TAG = "example";
```